plugins/inputs/wireless/README.md

```diff
@@ -29,10 +34,40 @@ The wireless plugin gathers metrics about wireless link quality by reading the `
     - misc (int64, packets, counter) - dropped for un-specified reason
     - missed_beacon (int64, packets, counter) - missed beacon packets
 
+### macOS metrics
+
+- metric
+  - tags:
+    - Interface (wireless interface, set to `airport`)
+    - State (running or not)
+    - Op mode (operating mode: station, ad_hoc or ap)
+    - 802.11 auth (open or hidden)
+    - link auth (authorization scheme)
+    - BSSID (mac address, _only_ reported if run as `root`)
+    - SSID (The SSID of the network)
+    - channel (channel number)
+  - fields:
+    - agrCtlRSSI (int64, dBm, gauge) - The current aggregate RSSI of the link
+    - agrExtRSSI (int64, dBm, gauge) - The current aggregate external RSSI of the link
+    - agrCtlNoise (int64, dBm, gauge) - The current aggregate noise of the link
+    - agrExtNoise (int64, dBm, gauge) - The current aggregate external noise of the link
+    - lastTxRate (int64, Mbps, gauge) - The last transmit rate
+    - maxRate (int64, Mbps, gauge) - The maximum transmit rate
+    - lastAssocStatus (int64, gauge) - The last association status
+    - MCS (int64, MCS, gauge) - The last MCS
+
 ## Example Output
 
 This section shows example output in Line Protocol format.
 
+### linux output
+
 ```shell
 wireless,host=example.localdomain,interface=wlan0 misc=0i,frag=0i,link=60i,level=-50i,noise=-256i,nwid=0i,crypt=0i,retry=1525i,missed_beacon=0i,status=0i 1519843022000000000
 ```
+
+### macOS output
+
+```bash
```

Comment:
Actually, `shell` isn't correct either, since the content in the code block is not a shell script. Best is to use `text` for influx line protocol data.

Reply:
So we're agreed that it should be `text`? I'll change it for the original as well as mine if we are all agreed here.